ui/src/main/kotlin/com/bitwarden/ui/platform/components/model/BitwardenOutlinedButtonColors.kt

```diff
@@ -0,0 +1,15 @@
+package com.bitwarden.ui.platform.components.model
+
+import androidx.compose.material3.ButtonColors
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.graphics.Color
+
+/**
+ * Colors for a [BitwardenOutlinedButton].
+ */
+@Immutable
+data class BitwardenOutlinedButtonColors(
```

Comment:
This is not new. It was extracted from `BitwardenOutlinedButton`. It seemed out of place embedded in `BitwardenOutlinedButton`.